frontend/src/app/auth.service.ts

```diff
@@ -47,9 +45,4 @@ export class AuthService {
         }
       });
   }
-
-  currentUser(): User {
```

Comment:
That method was removed as we use `changeUserEvent` instead?

Reply:
Now we will use `authService.user$` Observable.